lib/rubocop/cop/layout/space_around_method_call.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# TODO: when finished, run `rake generate_cops_documentation` to update the docs
+module RuboCop
+  module Cop
+    module Layout
+      # Checks dot method calls to not have spaces around them.
```

Comment:
I'd use "method call operator" terminology here.

Reply:
Makes sense. Changing it to `Checks method call operators to not have spaces around them.` 
Keeping it plural since it can have method chains also.